packages/flutter_migrate/lib/src/base/io.dart

```diff
@@ -25,6 +25,7 @@
 /// about any additional exports that you add to this file, as doing so will
 /// increase the API surface that we have to test in Flutter tools, and the APIs
 /// in `dart:io` can sometimes be hard to use in tests.
+library;
```

Comment:
Can we remove this and just make the comments  `//` instead? I'm not sure why they are `///` in the first place.

Reply:
Done